plugins/generator-1.19.4/forge-1.19.4/procedures/team_add_entity.java.ftl

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
Those additional braces outside are not needed as if already take care of wrapping inner variables, applies to most procedure blocks in the PR